setup-machine.sh

```diff
@@ -342,9 +356,22 @@ sudo rm -rf /lib/modules/*/kernel/drivers/net/*
 # delete any remembered existing network connections (e.g. wifi passwords)
 sudo rm -f /etc/NetworkManager/system-connections/*
 
-# set up the service for the selected machine type
-sudo cp config/${CHOICE}.service /etc/systemd/system/
-sudo chmod 644 /etc/systemd/system/${CHOICE}.service
+# Loop through the options using a for loop
+for service in "${CHOICES[@]}"
+do
+    if [ -f "config/${service}.service" ]; then    
+	sudo cp config/${service}.service /etc/systemd/system/
+	sudo chmod 644 /etc/systemd/system/${service}.service
+    fi
+done
+
+# watch and switch service
+sudo cp config/watch-and-switch.service /etc/systemd/system/
+sudo chmod 644 /etc/systemd/system/watch-and-switch.service
+sudo systemctl enable watch-and-switch.service
```

Comment:
so this will be running on scan too? Any potential issues with that?

Reply:
that's the one out-there risk I was mentioning... but I could just not do this on scan. Let me fix that.